appveyor.bat

```diff
@@ -318,23 +326,27 @@ copy /Y %DEPENDENCIES%\gettext64\libintl-8.dll       ..\runtime\GvimExt64\
 copy /Y GvimExt32\*.*                    ..\runtime\GvimExt32\
 copy /Y %DEPENDENCIES%\gettext32\libiconv-2.dll      ..\runtime\GvimExt32\
 copy /Y %DEPENDENCIES%\gettext32\libintl-8.dll       ..\runtime\GvimExt32\
-rem copy /Y c:\gettext32\libgcc_s_sjlj-1.dll ..\runtime\GvimExt32\
-rem libgcc_s_sjlj-1.dll is not needed anymore. Don't include it in the installer.
-if exist %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll del %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll
+@rem copy /Y c:\gettext32\libgcc_s_sjlj-1.dll ..\runtime\GvimExt32\
+@rem libgcc_s_sjlj-1.dll is not needed anymore. Don't include it in the installer.
+@rem if exist %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll del %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll
 copy /Y ..\..\diff.exe ..\runtime\
 copy /Y %DEPENDENCIES%\gettext%BIT%\libiconv-2.dll   ..\runtime\
 copy /Y %DEPENDENCIES%\gettext%BIT%\libintl-8.dll    ..\runtime\
-rem if exist c:\gettext%BIT%\libgcc_s_sjlj-1.dll copy /Y c:\gettext%BIT%\libgcc_s_sjlj-1.dll ..\runtime\
+@rem if exist c:\gettext%BIT%\libgcc_s_sjlj-1.dll copy /Y c:\gettext%BIT%\libgcc_s_sjlj-1.dll ..\runtime\
 copy /Y winpty* ..\runtime\
 copy /Y winpty* ..\..\
-set dir=vim%TAG_NAME:~1,1%%TAG_NAME:~3,1%
+set "dir=vim%TAG_NAME:~1,1%%TAG_NAME:~3,1%"
 mkdir ..\vim\%dir%
-xcopy ..\runtime ..\vim\%dir% /Y /E /V /I /H /R /Q
+xcopy ..\runtime ..\vim\%dir% /Y /E /V /I /H /R /Q /EXCLUDE:..\..\exclist.txt
 :: need to escape the ! because of delayedexpansion being active
-7z a ..\..\gvim_%TAG_NAME:~1%_%ARCH%.zip ..\vim -x^^!vim\%dir%\syntax\testdir\
+7z.exe a ..\..\gvim_%TAG_NAME:~1%_%ARCH%.zip ..\vim
 
 :: Create installer
-%CYGWIN_DIR%\bin\bash -lc "cd $(cygpath '%APPVEYOR_BUILD_FOLDER%')/vim/runtime/doc && touch ../../src/auto/config.mk && make uganda.nsis.txt"
+@rem %CYGWIN_DIR%\bin\bash -lc "cd $(cygpath '%APPVEYOR_BUILD_FOLDER%')/vim/runtime/doc && touch ../../src/auto/config.mk && make uganda.nsis.txt"
```

Comment:
> touch ../../src/auto/config.mk

I think this is still needed

Reply:
File "config.mk" it is used only in Unix-like systems. It is not required on Windows.
Now "nmake.exe" is used everywhere.

Reply:
I've just looked through the appveroy logs — everything is more or less normal.
Only for the 32‐bit version, there is one failure in the tests.

> Failures: 
	From test_options.vim:
	Found errors in Test_shortmess_F3():
	command line..script C:\projects\vim-win32-installer\vim\src\testdir\runtest.vim[607]..function RunTheTest[57]..Test_shortmess_F3 line 19: Expected ['bar'] but got ['foo']
TEST FAILURE 
NMAKE : fatal error U1077: 'if exist test.log ( echo TEST FAILURE & exit /b 1 )  else ( echo ALL DONE )' : return code '0x1'
Stop

Reply:
And this is also for the 64‐bit version.

> highlight.c(882): warning C4267: '+=': conversion from 'size_t' to 'int', possible loss of data

Reply:
unrelated issue, should have been fixed by: https://github.com/vim/vim/commit/72a156b4edfdf1502d421370cf38281c258e3c4a